examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -52,24 +55,30 @@ const replaceOptions = {
 }
 
 const workboxOrInjectManifestEntry = {
-	// vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
+	// Vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
```

Comment:
I didn't understand this comment until you shared more details on Discord. Could be helpful to add those details here
```suggestion
	// Vite generates <name>.<hash>.<ext> layout whlile SvelteKit generates <name>-<hash>.<ext
	// Vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
```